scripts/compile-mlibc-sysdeps.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+SYSDEPS=${3:-"dripos lemon aero ironclad vinix lyre"}
```

Comment:
it's prob better to just pop the non-sysdeplist arguments first and use $@ as the list of subsysdeps

Reply:
This doesn't really work, as `$@` is an array IIRC. Maybe this is better?
```bash
SYSDEPS=${*:-"dripos lemon aero ironclad vinix lyre"}
```
The shifting doesn't help there, too, but is nice for the sake of clarifying what `$1` and `$2` mean.

Reply:
SYSDEPS is redundant